ui/desktop/app/components/settings-card/preferred-clients/index.hbs

```diff
@@ -0,0 +1,60 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<SettingsCard
+  @header={{t 'settings.preferred-clients.title'}}
+  @icon='external-link'
+  @description={{t 'settings.preferred-clients.description'}}
+>
+  <:body>
+    <Hds::Table
+      class='full-width'
+      @model={{this.protocolClients}}
+      @columns={{array
+        (hash
+          label=(t 'settings.preferred-clients.table.header.protocol')
+          width='50%'
+        )
+        (hash
+          label=(t 'settings.preferred-clients.table.header.client') width='50%'
+        )
+      }}
+    >
+      <:body as |B|>
```

Comment:
I noticed that `none detected. we recommend windows app` option from figma is missing. should we add support for that option here?